### PR TITLE
Build fixes

### PR DESCRIPTION
**Pull request by hadess**
Small build fixes for problems I ran into trying to build the repo today.

### CODE REVIEW

File: enchant.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ config-opts:
   - --with-myspell-dir=/app/share/myspell
 sources:
   - type: archive
-    url: https://www.abisource.com/downloads/enchant/1.6.0/enchant-1.6.0.tar.gz
+    url: https://src.fedoraproject.org/lookaside/extras/enchant/enchant-1.6.0.tar.gz/de11011aff801dc61042828041fb59c7/enchant-1.6.0.tar.gz
```
Comment (hfiguiere) [inline enchant.yaml:8]: you could also use the 1.6.1 tarball from the Github releases (official)

https://github.com/AbiWord/enchant/releases/tag/enchant-1-6-1
https://github.com/AbiWord/enchant/releases/download/enchant-1-6-1/enchant-1.6.1.tar.gz

**Comment by hfiguiere** [comment]
bot, build

**Comment by hadess** [comment]
Looks like the build failed on an infrastructure problem rather than one with the code changes.

**Comment by hfiguiere** [comment]
yeah. There is ongoing changes on the infra. It's also why I wanted a build before merging.

**Comment by hfiguiere** [comment]
bot, build

**Pull request merged**